decidim-admin/config/locales/ca.yml

```diff
@@ -31,9 +34,12 @@ ca:
         dashboard: Tauler de control
         participatory_processes: Processos participatius
         participatory_processes_submenu:
+          attachments: Adjunts
           info: Informació
           process_admins: Administradors del procés
+          settings: Ajustos
```

Comment:
Ajustos? "Configuració" is more widespread, I think. TERMCAT [suggests "Configuració"](http://www.termcat.cat/) (try searching for `settings`), Softcatalà [does too](https://www.softcatala.org/recull-termes-softcatala/).